modules/engines/engine-hdb/src/main/java/com/sap/xsk/hdb/ds/transformer/hdbdd/HdbddTransformer.java

```diff
@@ -169,16 +170,7 @@ public String traverseSelectStatements(ViewSymbol viewSymbol, List<String> alias
 
       // Check if the dependant table has :: to know whether short or full name is used in the hdbdd view definition. In case it is not we should build the full name
       if (!dependsOnTable.contains(PACKAGE_DELIMITER)) {
-        // Check if the dependant table name is DUMMY. This is a reserved table name for hana dummy tables. We make sure to make it in uppercase
-        if (dependsOnTable.equalsIgnoreCase(DUMMY_TABLE)) {
-          dependsOnTable = dependsOnTable.toUpperCase();
-        } else {
-          String dependsOnTableFullName = fullTableNameBuilderFromViewSymbol(dependsOnTable, viewSymbol);
-          // Replace the short name in the select columns with the full name
-          selectColumns = replaceWithQuotes(selectColumns, dependsOnTable, dependsOnTableFullName);
-          // Set the dependant table to be with the full name
-          dependsOnTable = dependsOnTableFullName;
-        }
```

Comment:
why did you delete this logic? What about the cases that are missing now?

Reply:
This was moved to a private method called `getFullTableName`

Reply:
but what about line 178?